src/private/meetingRoom.ts

```diff
@@ -0,0 +1,245 @@
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError } from '../public/interfaces';
+
+export namespace meetingRoom {
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent a meeting room.
+   */
+  export interface MeetingRoomInfo {
+    /**
+     * Endpoint id of the meeting room.
+     */
+    endpointId: string;
+    /**
+     * Device name of the meeting room.
+     */
+    deviceName: string;
+    /**
+     * Client type of the meeting room.
+     */
+    clientType: string;
+    /**
+     * Client version of the meeting room.
+     */
+    clientVersion: string;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate meeting room capabilities.
+   */
+  export enum Capability {
+    /**
+     * Media control capability: toggle mute.
+     */
+    TOGGLE_MUTE = 'toggleMute',
+    /**
+     * Media control capability: toggle camera.
+     */
+    TOGGLE_CAMERA = 'toggleCamera',
+    /**
+     * Media control capability: toggle captions.
+     */
+    TOGGLE_CAPTIONS = 'toggleCaptions',
+    /**
+     * Media control capability: volume ajustion.
+     */
+    VOLUME = 'volume',
+    /**
+     * Stage layout control capability: show gallery mode.
+     */
+    SHOW_VIDEO_GALLERY = 'showVideoGallery',
+    /**
+     * Stage layout control capability: show content mode.
+     */
+    SHOW_CONTENT = 'showContent',
+    /**
+     * Stage layout control capability: show content + gallery mode.
+     */
+    SHOW_VIDEO_GALLERY_AND_CONTENT = 'showVideoGalleryAndContent',
+    /**
+     * Stage layout control capability: show laryge gallery mode.
+     */
+    SHOW_LARGE_GALLERY = 'showLargeGallery',
+    /**
+     * Stage layout control capability: show together mode.
+     */
+    SHOW_TOGETHER = 'showTogether',
+    /**
+     * Meeting control capability: leave meeting.
+     */
+    LEAVE_MEETING = 'leaveMeeting',
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent capabilities of a meeting room.
+   */
+  export interface MeetingRoomCapability {
+    /**
+     * Media control capabilities, value can be "toggleMute", "toggleCamera", "toggleCaptions", "volume".
+     */
+    mediaControls: string[];
+    /**
+     * Main stage layout control capabilities, value can be "showVideoGallery", "showContent", "showVideoGalleryAndContent", "showLargeGallery", "showTogether".
+     */
+    stageLayoutControls: string[];
+    /**
+     * Meeting control capabilities, value can be "leaveMeeting".
+     */
+    meetingControls: string[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate meeting room state of stage layout.
+   */
+  export enum StageLayoutState {
+    /**
+     * Stage layout in gallery mode.
+     */
+    GALLERY = 'Gallery',
+    /**
+     * Stage layout in content + gallery mode.
+     */
+    CONTENT_GALLERY = 'Content + gallery',
```

Comment:
Please stick with indexed enums or use strings that match the enum values, e.g.:
```
ContentGallery = 'ContentGallery',
Content = 'Content',
LargeGallery = 'LargeGallery,
```

Reply:
Changed to upper camel case as well. For 'Content + gallery', it will keep like this, since the string will show to user and it is by design.